tools/update_metainfo.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import datetime
+import os
+import re
+
+import markdown
+import bs4
+from lxml import etree
+
+
+def parse_changelog(content):
+    for section in re.split("^## ", content.strip(), flags=re.MULTILINE)[1:]:
+        version, sep, description_md = section.partition("\n")
+        matchobj = re.match(
+            r"\[(?P<number>\d\.\d\.\d)\]\([^\)]+\)"
```

Comment:
I struggle to understand the `[^\)]+` part.

Reply:
I recommend regex101.com: 
![Screenshot from 2021-09-08 17-08-07](https://user-images.githubusercontent.com/1834516/132536125-60781040-ee1f-4968-95c5-bc12508567bb.png)

It's supposed to match any char except `)` one or more times (for the milestone URL):
![Screenshot from 2021-09-08 17-13-14](https://user-images.githubusercontent.com/1834516/132536959-59d33183-7e98-42cf-a9af-d49de00504ca.png)